### DISCUSSION TITLE
大家有什麼推薦的樂團嗎？

### DISCUSSION THREAD

**Discussion by nelson910402** [1101-GS4719-C/Python-Community-GS4719-C#1930] [Q&A]
如題。我個人喜歡搖滾、饒舌類的，偶爾也會聽復古一點的歌。最近在Netflix看了《波希米亞狂想曲》，所以比較迷皇后樂團，像《I wanna break free》之類的就很洗腦。台灣樂團則是比較喜歡珂拉琪，他們的歌詞設計的很好，我很享受一邊聽音樂一邊分析故事的感覺。

另外問一下卡卡的掛，人不在桃園所以沒聽到，低卡上好多人在罵，到底是有多吵？

**Comment by EPHMR0524** [comment]
日本樂團大推羊文学，マヨイガ和天気予報我覺得都很頂，聽起來感覺如何我自己不太會描述，總之就是非常好聽。
至於最近卡卡音樂祭的爭議的話我覺得應該是噪音問題，畢竟不是所有在學校的人都聽團，自然會有人出來抗議，吵的程度大概是在電機系館能聽到鼓聲的程度。

**Reply by woohsin** [reply]
所以假日知道有音樂季我就逃回家了！辛苦大家了...

**Comment by SusuTima** [comment]
很吵很吵，吵到耳朵快爛掉的那種。但是坐在宿舍有免費的怕胖團可以聽，想想就覺得算了。最後推薦去聽怕胖團的《後照鏡》，是很冷門但是很好聽的一首歌。

**Reply by woohsin** [reply]
我雖然回家，但我是有說中午12.就會被起床聽團;)

**Comment by tzurou** [comment]
我個人很喜歡聽胡凱兒，他的曲風很Emo，歌詞也寫得很好超適合Emo時聽，推薦去聽他們去年發的專輯《天色》，整張專輯連貫有故事性，也展現了他們自己所定義的「情緒搖滾」。另外我也很喜歡粗大Band，是很熱血的龐克曲風，聽他們的歌都會很有正能量。
P.S. 四月底的上岸音樂節有這兩團！！我一定去聽爆！！

**Comment by tshsu04224** [comment]
韓國樂團的話推薦n flying，他們2024跨年演唱會的時候也有來桃園唱，曲風很多元，有偏搖滾的、抒情的，還有青春類型（？他們也常常會翻唱中文、英文或日文等歌曲，反正實力蠻強的，我個人很喜歡聽他們的歌，推一首rooftop，是我聽一次就回不去的那種

**Reply by woohsin** [reply]
他們鼓手跟主唱演的那個小短劇超好笑，很推薦！

**Comment by woohsin** [comment]
美國的LANY，他們已經連續來台灣開演唱會三年了。一樣的是我都沒搶到票，高中考學測時就是把他們第一張專輯拿出來聽，有種懶懶的、輕鬆說事情的感覺，也有比較活潑的歌，常常會被拿去翻唱(我入坑就是聽韓國樂團 the rose翻唱他們的ILYSB)

**Comment by woohsin** [comment]
日本的話真心推薦OOR，(雖然他們現在被戲稱one ok pop)，其實他們開始用電子樂器之後我也有牴觸過一陣子(不是不喜歡電子，只是很喜歡他們之前那種風格)
其實我聽他們的歌時基本不會看歌詞，光聽就有種很熱血、年輕的感覺，如果看歌詞，那是真的滿正向的，適合叛逆期青少年 :)
還有RADWIMPS(他們也是我搶票兩年都沒有成功，喔我OOR也沒搶到，超爛)很多人都沒聽過他們，但是說新海誠電影的配樂大家就懂了。
(題外話，新海誠我真的不懂，畫面美歸美，但劇情我每次都母颯颯。所以，細田守推推!!)

**Reply by woohsin** [reply]
更)我有搶到dept的票了，雖然是別人幫我的。

**Reply by 110101501** [reply]
OOR真的很燃…上次來臺開唱的時候朋友有去，超級羨慕

**Comment by woohsin** [comment]
台灣的坷拉琪真的很棒，多種語言一次享受，而且歌詞真的很有故事感，之前yt 有頻道在分析他們的歌詞，看完之後發現故事其實有點沈重，音樂也滿重，所以晚上聽是真的很適合！

**Comment by yun1007** [comment]
夏日入侵企劃我覺得有幾首還不錯，例如極惡都市歌詞很中二我就很喜歡哈哈哈，想去海邊應該是最多人聽過的。回春丹主唱聲線狠抓人耳朵。

**Comment by sunset513** [comment]
我也很喜歡皇后樂團，特別是 love of my life ，浪漫的旋律搭配酸楚的歌詞，這股矛盾感總是激起我的心中的波紋。

我這裡想要推薦的樂團是經典中的經典 The Beatles

他們是音樂史上的巨人，音樂是跨越了世代，深深地觸動了我的心靈。像是《Hey Jude》那份深情的力量，還是《Let It Be》的安慰，都讓我們感受到音樂的魔力，給與我繼續奮鬥的力量。對我來說，The Beatles不僅是樂隊，更是一種信仰。

**Comment by wanka2030** [comment]
不知道韓團算不算ㄟ最近真的非常hot(G)i-dle無限loop瘋狂洗腦，成軍6年以來每每回歸都帶給neverland有著不同的感覺，(G)I-DLE剛出道時，最讓人印象深刻的就是編曲和MV都充滿特殊異域民族風情，從出道曲〈LATATA〉的西域風、〈Senorita〉的拉丁美洲風、〈Uh-Oh〉的西部牛仔、〈Lion〉的中世紀美洲貴族風及神秘宗教色彩的〈Oh My God〉，(G)I-DLE的作品就像是藝術品，MV美到屏息配合著強烈、迷幻的旋律，只要看過一次就會深刻記在腦海中。

**Comment by thomasxshen** [comment]
最近在聽夏日入侵計畫，想去海邊很好聽。回春丹的鮮花也是膾炙人口，雖然最近被民主鐵拳給痛毆了一頓，但我還是很推薦大家去聽。

**Comment by 110101501** [comment]
我很喜歡來自日本的King Gnu…當初因為《白日》這首歌認識他們，井口理的聲音聽起來很唯美很舒服。而他們所有歌曲中我最喜歡《飛行艇》，有種在逆境中重生、永不畏懼、揮灑生命的感覺。尤其在他們來台灣開演唱會時，把這首歌放在安可曲，直接讓現場嗨翻。
此外，他們還因為0403的大地震，將臺灣場兩場演唱會的收益全數捐出，超級霸氣，很佩服！

**Comment by Taoyi-Chiang** [comment]
推薦法國樂團Indo-chine，很適合在下午傍晚的時候一邊聽一邊耍廢吃零食發呆這樣。
也推薦羅馬尼亞電音組合Fly Project，充滿拉丁風情，很適合一邊聽一邊跟著搖。
如果想要比較有空靈的話Giolì & Assia感覺人聲處理是北歐風格，說到北歐樂團也要推薦一下Röyksopp，整個聽起來就像是日出照在融冰一般的感受。

**Comment by Boying12** [comment]
搖滾樂團我推薦Queen ，作為經典搖滾樂團，Queen的音樂風格多樣，從搖滾到歌劇都有涉獵，經典歌曲如《Bohemian Rhapsody》和《We Will Rock You》是必聽的。此外，還有Linkin Park ， 這個樂團結合了搖滾、嘻哈和電子元素，Linkin Park的音樂風格非常獨特，像《In the End》和《Numb》都深受粉絲喜愛。

**Comment by Tchang323** [comment]
超級推One Direction，雖然很不幸的是，他們有一個團員Liam在上個月過世了嗚嗚 但他們的歌真的都百聽不膩!

**Reply by kzx-creat** [reply]
推，1D是我的青少年回憶啊!!!! (我問現在的國高中生，他們好像都不太認識這團，這是時代差嗎 LOL

**Comment by Tchang323** [comment]
很喜歡告五人的音樂，旋律很catchy，一直很想去聽它們的演唱會

**Comment by CeBru13** [comment]
非常推薦 BABYMETAL，每個人都應該聽看看。
她們是重金屬，同時也是可愛妹子，所以她們是可愛金屬
(雖然演唱會現場看起來很像邪教，
可以去youtube上找

**Comment by ShuliANN** [comment]
我個人很喜歡聽胡凱兒和Control T，胡凱兒他們的音樂主要對生活和心靈有細膩體會，而且曲風、旋律多樣且具層次感，從柔情到爆發力兼具，尤其在現場表演中展現出對音樂的深厚情感。而Control T他們的音樂都給人一種溫柔、治癒的感覺，聽了會放鬆心情。

**Comment by kzx-creat** [comment]
大推韓國的wave to earth，曲風偏慵懶，我可以整天循環撥放他們的專輯。而且歌詞蠻多都是英文的，不太會有看不懂韓文也找不到中翻的問題。明年他們要來台灣，可以去。

**Comment by Teddy111802518** [comment]
日本的 one ok rock 已經很老了 但是我覺得他的歌詞真的很給年輕人力量 如果覺得人生徬徨 不妨去聽聽他們的歌 有故事的人可能會哭 這邊推薦 We are

**Reply by Teddy111802518** [reply]
還有11/28要來的imagine dragon 話說我搶道票了超扯 demons再描述與自己心魔的鬥爭 希望大家都能成為自己的believers

**Comment by Teddy111802518** [comment]
還有11/28要來的imagine dragon 話說我搶道票了超扯 demons再描述與自己心魔的鬥爭 希望大家都能成為自己的believers

**Comment by forestdew507** [comment]
很喜歡蘇打綠 ! 蘇打綠的音樂有些朗朗上口、很直白就可以領會其中的概念，有些歌曲又不好預期、可以反覆咀嚼字句去詮釋出自己的版本 ( 解讀總是很主觀的歌曲更容易在不同情境中聽時帶走不一樣的感受 ) 而且他們的詞多數寫得很好，甚至會讓人很慶幸自己的母語就是華語才可以感受到一些也許身為母語人士才能感受的歌曲氛圍 !

**Comment by yuan3x2** [comment]
我自己超愛一個澳大利亞的樂團Five Second of Summer，他們的歌都很讚，也不會太口水，我自己聽也不太容易膩，而且各種風格的歌他們幾乎都有，可惜的是他們的世界地圖中沒有台灣( ´•̥̥̥ω•̥̥̥` )。
此外我還很喜歡一個英國的樂團The Vamps（都喜歡老團），主唱的聲音真的超讚。

**Comment by forestdew507** [comment]
前陣子因為《影后》找來小男孩樂團唱插曲片尾，讓我又挖了小男孩的歌回來聽。遲鈍的驚喜發現其實他們的第一張專輯《everything》早就已經在我的記憶裡。

蠻好聽的，無論是主唱的聲線還是團長的作詞，首推〈我的親愛小孩〉，這首歌的概念後勁蠻強的 ! 

( 後續 ) 不過瘋狂聽了三五天，沒想到就有點倦了XD 但我覺得就創作來說是值得一聽的樂團，有些歌曲的概念我很喜歡:) 首張專輯做得很好。一算明年就是小男孩十周年了，雖然因為耳朵倦了 ? 所以不算成功入坑不過依然期待新作

**Comment by christaaah** [comment]
雖然不太聽樂團，但是偶爾在隨機歌單的時候會發現衣些好歌，所以認識了一些樂團。日本的Official鬍子男dism，他們的歌都蠻好聽的，現場也非常厲害，下星期就是他們在台灣的演場會了，另外最近也發現Mrs. GREEN APPLE的歌也很好聽。
另外雖然忘記是哪個，但是在中正圖書館附近草地的那個是真的吵，而且時間持續很久，在宿舍一整天都聽得到，之後搬到棒球場附近的宿舍就比較聽不到了，算個人還能忍受程度，雖然就是換個地方吵了。
